examples/tutorial_dbt_dagster/tutorial_finished/tutorial_dbt_dagster/repository.py

```diff
@@ -1,8 +1,7 @@
 import os
```

Comment:
the code snippets for the dbt tutorial are hard-coded into the tutorial mdx page (since there isn't a good way to do incremental updates to the same code snippet). There's a section that creates a duckdb io manager in `docs/content/integrations/dbt/using-dbt-with-dagster/part-three` that should be updated